### PR TITLE
Change pubsub method to redis

### PR DESCRIPTION
**Pull request by Lacrymology**
Depends on https://github.com/inaes-tic/mbc-common/pull/6

### CODE REVIEW

**Comment by Lacrymology** [comment]
Also, I haven't yet finished the caspa side of things

File: test/playlist-mongo-driver.js [hunk 1]
```diff
@@ -99,18 +99,19 @@ describe('PlaylistMongoDriver', function(){
             self.pubsub = mbc.pubsub();
 
             self.message = {
-            backend: 'schedbackend',
-            model: {
-                start: moment(new Date()).unix(),
-                end: moment(new Date()).add(5*60*1000).unix(),
+                backend: 'schedbackend',
+                model: {
+                    start: moment(new Date()).unix(),
```
Comment (xaiki) [inline test/playlist-mongo-driver.js]: you don't need to pass new Date() to moment.
just use moment().unix()

Comment (Lacrymology) [inline test/playlist-mongo-driver.js]: ah, yes, this line was from before I discovered that

Comment (jmrunge) [inline test/playlist-mongo-driver.js]: Can you change it please? Its more clear the way @xaiki pointed...

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
         "watchr"      : ">=2.3",
         "moment"      : "2.0.0",
         "mubsub"      : "git://github.com/inaes-tic/mubsub.git",
-        "mbc-common"  : "git://github.com/inaes-tic/mbc-common.git",
+        "mbc-common"  : "git://github.com/inaes-tic/mbc-common.git#2.0.0",
```
Comment (jmrunge) [inline package.json]: Can you remove this and work with me to merge both PR together?

Comment (Lacrymology) [inline package.json]: as I said, I don't really understand why is this bad. I rather think we
should ALWAYS have the version tagged

**Pull request merged**